### PR TITLE
Fix for #5692

### LINKED ISSUE #5692: Temperature input plugin stops working when WiFi is turned off
### Relevant telegraf.conf:

[[inputs.temp]]

### System info:

- Telegraf 1.10.1-1 installed from .deb package / Ubuntu 16.04 + Kernel 5.0
- Intel Wireless LAN hardware, iwlwifi module

### Steps to reproduce:

1. Enable WiFi, connect to a network
2. Enable the `inputs.temp` plugin
3. Observe that the plugin is working correctly: various `temp` metrics should exist, including one called `temp,sensor=iwlwifi_input`
4. Disconnect from the network, turn off WiFi

### Expected behavior:

No new metrics should be produced for `temp,sensor=iwlwifi_input`. New metrics should still be produced for everything else in `temp`.

### Actual behavior:

**No** new metric is produced for anything in `temp`.

### Additional info:

    Apr 08 14:36:50 telegraf[6305]: 2019-04-08T06:36:50Z E! [inputs.temp]: Error in plugin: error getting temperatures info: read /sys/class/hwmon/hwmon4/temp1_input: no data available

### PR DESCRIPTION
**Pull request by marcv81**
In case of non-critical temperature warnings: silently ignore them instead of dropping all the temperature metrics. I would appreciate friendly advice from the maintainers as to how to log the warnings instead please. I'm not sure whether to log them directly from Temperature() (probably not!) or to pass them anywhere else for logging.

closes #5692

### Required for all PRs:

- [X] Signed [CLA](https://influxdata.com/community/cla/).
- [X] Associated README.md updated.
- [X] Has appropriate unit tests.

### CODE REVIEW

**Comment by marcv81** [comment]
Would appreciate some kind help with the CI tests please.
- To update `shirou/gopsutil` I ran `dep ensure -update github.com/shirou/gopsutil` and committed the resulting changes. How should I have approached this?
- I do not believe I modified `plugins/inputs/ping` or any of its dependencies. Am I wrong?
- I was able to run `make package` locally (Ubuntu Bionic), install the package, and no smoke came out.

**Comment by danielnelson** [comment]
I think you did the `dep ensure` step correctly, but it looks maybe like your IDE reformatted some files for you automatically.  You will need to reset these files back to the unmodified versions (including Go.lock) so that it is only the temperature changes.

**Comment by marcv81** [comment]
The issue was on outdated version of dep. Thanks for your comment which pointed me in the right direction @danielnelson. Also FYI I had to push the same change twice because the first build failed AppVeyor due to network issues.

**Comment by danielnelson** [comment]
Can you tell me about the cpu stolen changes?

**Comment by marcv81** [comment]
@danielnelson, gopsutil removed cpu.Stolen because it was declared but never set. The proposed change is required to update gopsutil as they introduced a breaking API change. The commit links to a PR in gopsutil, which in turns links to an issue with more information.

File: plugins/inputs/elasticsearch/README.md [hunk 1]
```diff
@@ -219,7 +219,6 @@ Operating system stats, load average, cpu, mem, swap measurement names:
   - cpu_user value=0
   - cpu_idle value=99
   - cpu_usage value=0
-  - cpu_stolen value=0
```
Comment (danielnelson) [inline plugins/inputs/elasticsearch/README.md]: I think this shouldn't be changed, these OS stats come from the Elasticsearch server and not gopsutil.

Comment (marcv81) [reply plugins/inputs/elasticsearch/README.md]: Well spotted, thanks for the careful review. Updated accordingly.

**Pull request merged**